acceptance-tests/tests/commands/projectCreate.spec.ts

```diff
@@ -23,13 +23,12 @@ describe('hs project create', () => {
   });
 
   it('should create a project containing a private app', async () => {
-    await testState.cli.execute([
+    await testState.cli.executeWithTestConfig([
```

Comment:
Here's an example of us calling the new cli method `executeWithTestConfig` to prevent us from having to pass in the config override argument.